include.js

```diff
@@ -2037,6 +2037,13 @@ let cards = [
     imageLink: './Art/SamilAbud/bouncing-effect.gif',
     author: 'Samil Abud',
     githubLink: 'https://github.com/samilabud'
+  },
+  {
+    artName: 'Half Spin Points',
+    pageLink: './Art/ParzivalAFK/HalfSpinPoints.html',
+    imageLink: './Art/ParzivalAFK/HalfSpinPoints.gif',
```

Comment:
That gif does not exist in your contribution. The file you commited is called `Animation1.gif`. You can either rename the file, or the link to the file.

Reply:
Great catch, thank you so much!